### PR TITLE
added create_elbow_plot function

### PR DESCRIPTION
**Pull request by JialuJin**

### CODE REVIEW

**Comment by davisli1** [comment]
@JialuJin  can you please merge conflicts first?

**Comment by JialuJin** [comment]
merged @davisli1

File: tests/testthat/create_elbow_plot_test.R [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+training <- sample_n(data, nrow(data)*0.5, replace = FALSE)
+results <- find_k_min(training)[[4]]
+
+# Test that the output image has correct title
+test_that("`create_elbow_plot` should have title 'K Neighbours vs RMSE'", {
+  elbow_plot <- create_elbow_plot(results)
+  expect_equal(elbow_plot$label$title, "K Neighbours vs RMSE")
+})
+
+test_that("`create_elbow_plot` should throw an error when incorrect types
```
Comment (davisli1) [inline tests/testthat/create_elbow_plot_test.R:15]: Could you please add an annotation for the last test to describe what it does?

Comment (JialuJin) [reply tests/testthat/create_elbow_plot_test.R:15]: I will fix that for all test files when I make another pull request

**Comment by davisli1** [comment]
LGTM

**Pull request merged**